internal/core/commands/create_device_settings.go

```diff
@@ -20,15 +20,7 @@ func NewCreateDeviceSettingsCommandHandler(dbs func() *db.ReaderWriter) CreateDe
 }
 
 type CreateDeviceSettingsCommandRequest struct {
-	ID                                     int64
-	TemplateName                           string
-	BatteryCriticalLevelVoltage            float64
-	SafetyCutOutVoltage                    float64
-	SleepTimerEventDrivenInterval          float64
-	SleepTimerEventDrivenPeriod            float64
-	SleepTimerInactivityAfterSleepInterval float64
-	SleepTimerInactivityFallbackInterval   float64
-	WakeTriggerVoltageLevel                float64
+	TemplateName string
```

Comment:
same here, think we're missing a property fro the settings json content

Reply:
here I added SettingsData struct to handle null.JSON settings field